ETC.tsv

```diff
@@ -19582,171 +19582,171 @@ ETC_20151224_019583
+ETC_20151224_019666	UI mode : Fill
+ETC_20151224_019667	Target Info
+ETC_20151224_019668	Change Target
+ETC_20151224_019669	ESC Button
+ETC_20151224_019670	UI Selection Mode Change
+ETC_20151224_019671	Target Lock
+ETC_20151224_019672	Move Up
+ETC_20151224_019673	Move Down
+ETC_20151224_019674	Move Left
+ETC_20151224_019675	Move Right
+ETC_20151224_019676	Move Up-Left
+ETC_20151224_019677	Move Up-Right
+ETC_20151224_019678	Move Down-Left
+ETC_20151224_019679	Move Down-Right
+ETC_20151224_019680	Pet Companion
+ETC_20151224_019681	Companion Info
+ETC_20151224_019682	Rest/Sit
+ETC_20151224_019683	Select/Skip
+ETC_20151224_019684	Chat Change
+ETC_20151224_019685	D-pad Up
+ETC_20151224_019686	D-pad Down
+ETC_20151224_019687	D-pad Left
+ETC_20151224_019688	D-pad Right
+ETC_20151224_019689	TP Supply/Payment(Minus )(Trial Time: above item rules apply)
+ETC_20151224_019690	TP Supply/Payment(Minus )
+ETC_20151224_019691	EX2> WIZ:M/WAR:F (Wizard Male, Swordsman Female only Supply)
+ETC_20151224_019692	EX1> WIZ/WAR (Wizard, Swordsman only Supply)
+ETC_20151224_019693	Fetching Stat Usage Data...
+ETC_20151224_019694	Fetching Skill Usage data...
+ETC_20151224_019695	Fetching TP Usage data...
+ETC_20151224_019696	Data Read-in was exceeded, thus request was canceled. {nl} Please inquire management
+ETC_20151224_019697	Requested incorrect history information. {n} Please inquire management
+ETC_20151224_019698	Exceeded Market Goods Registration Count
+ETC_20151224_019699	Initialization Count
+ETC_20151224_019700	Use {TP} TP to purchase Character Slots?
+ETC_20151224_019701	TP Cost
+ETC_20151224_019702	Nexon PC Cafe
+ETC_20151224_019703	Market Sales Tax
+ETC_20151224_019704	Market Goods Registration Count Increase
+ETC_20151224_019705	Simultaneous Attribute Training Count Increase
+ETC_20151224_019706	The following feature cannot be used.
+ETC_20151224_019707	{Target} Character cannot trade.
+ETC_20151224_019708	Trading Available
+ETC_20151224_019709	EXP Increase from defeated monsters
+ETC_20151224_019710	Stamina Recovery Rate Increase
+ETC_20151224_019711	Market Goods Registration Count {COUNT} Increase
+ETC_20151224_019712	Simultaneous Attribute Training Count {COUNT} Increase
+ETC_20151224_019713	Movement Speed +{COUNT} Increase
+ETC_20151224_019714	The token item is already activated.
+ETC_20151224_019715	Premium goodies have arrived in your Lodge.{nl}Please check them at your Lodge.
+ETC_20151224_019716	 the item
+ETC_20151224_019717	Character
+ETC_20151224_019718	Server Team
+ETC_20151224_019719	This item will bind to you when purchased.{nl}Are you sure you want to purchase?
+ETC_20151224_019720	Are you sure you want to{nl}put the following item in the Market?
+ETC_20151224_019721	Following item(s) cannot be sold to NPCs
+ETC_20151224_019722	Stamina Recovery Rate 2x Increase
+ETC_20151224_019723	Free Admission Count have been met.Use {TP} TP to enter {INDUN}?{nl}You may only enter dungeons that are already activated without TP charge.
```

Comment:
You've used up all of your free entry counts.{nl}Do you want to spend {TP} TP to enter {INDUN}?{nl}However, you may only enter dungeons that are already activated without the consumed TP.